gbus/metrics/saga_metrics.go

```diff
@@ -20,7 +20,7 @@ var (
 		Subsystem: "saga",
 		Name:      "latency",
 		Help:      "The latency of the entire saga",
-	}, []string{"Service", "SagaType"})
+	}, []string{"Service", "SagaType", "Initiator"})
```

Comment:
@kerenbe4 why are we adding the initiator as the label and not the service that is running the saga ?
there can be a lot of initiators for a single type of saga.

Reply:
never mind, won't harm having that data as well.